### PR TITLE
Fix repository to ensure that runs on dataset

### PR DESCRIPTION
**Pull request by cyrilyared**

### CODE REVIEW

**Review by mauriciolp — COMMENTED** [review]
Thank you for the contribution.
Just a few notes about the suggested changes:

- In the config files (.cfg), I left the "weights_filepaths" variable with a placeholder value (".../fold_x/rerun_x/..."), because run_protocol.py will replace this value during its main loop, through function "find_best_weights".
- The replacement of 'acc' by 'accuracy' is required for newer version of keras + tensorflow (>2.0?).  For tensorflow v1.14, it should still be 'acc', if I am not mistaken.

**Comment by cyrilyared** [comment]
Thanks for the feedback. I realized I created a pull-request to your repository instead of our forked repository that we are working with. 

- I think as well that the replacement of 'acc' by 'accuracy' if for tensorflow 2.0. 
- Thanks for letting me know about the .cfg file, I didn't realize it would replace the value during the main loop but this functionality is very useful.
 
I will close this pull request but nevertheless, thanks for your help!
